bundle/edu.gemini.model.p1/src/main/scala/edu/gemini/model/p1/overheads/Overheads.scala

```diff
@@ -31,11 +31,11 @@ object Overheads extends (BlueprintBase => Option[Overheads]) {
       ObservationTimes(TimeAmount(progTimeHrs, TimeUnit.HR), TimeAmount(partTimeHrs, TimeUnit.HR))
     }
   }
-  private case class TelluricOverheads(telluricTime: TimeAmount) extends Overheads {
-    val VisitLength = 1.5
+  private case class ScienceHoursPerTelluric(telluricTime: TimeAmount) extends Overheads {
+    val VisitLength = TimeAmount(1.5, TimeUnit.HR)
```

Comment:
Oh, it was actually the `VisitLength` name itself I was confused about.  Maybe I'm just generally confused though.

Reply:
😄 Te name `VisitLengh` comes directly from the original ticket

Reply:
Ok, I don't think they care what we call our internal variables but maybe it's best to match what they put in the task.  I think I'd put the class name back to `TelluricOverheads` though.  Sorry for the hassle!  It looks good. :+1:

Reply:
`NIRSpectroscopyOverheads` seems good.